ddgs/engines/duckduckgo_news.py

```diff
@@ -44,7 +44,16 @@ def build_payload(
             "p": safesearch_base[safesearch.lower()],
         }
         if timelimit:
-            payload["df"] = timelimit
+            if _is_custom_date_range(timelimit):
+                # Handle custom date range: YYYY-MM-DD..YYYY-MM-DD
+                date_range = _parse_date_range(timelimit)
+                if date_range:
+                    start_date, end_date = date_range
+                    # DuckDuckGo News uses after:YYYY-MM-DD before:YYYY-MM-DD in query
+                    payload["q"] += f" after:{start_date} before:{end_date}"
```

Comment:
time range is not supported in duckduckgo news.
But presumably, it can work with the same parameters as in the duckduckgo text.